docs/GettingHelp.md

```diff
@@ -7,5 +7,5 @@ Please note that despite the `wdl` currently in its name, this is a forum that s
 **Website and User Guide**
 
 The [WDL website](https://software.broadinstitute.org/wdl/) is the best place to go for more information on WDL.
-In particular new users should check out the [user guide](https://software.broadinstitute.org/wdl/userguide/)
+In particular new users should check out the [openWDL WDL docs (wdl-docs)](https://wdl-docs.readthedocs.io/en/stable/)
```

Comment:
This site looks great! How have I never heard of it!?

Reply:
Thanks @aednichols! We just added it. We moved the Terra WDL docs over to OpenWDL as part of a larger push to get the community to contribute to wdl documentation. The idea is that anyone can contribute to the repo by forking and putting in a PR. We're still updating the contribution guidelines and adding final touches to the Cookbook-related docs.